app/utils/localAuthentication.ts

```diff
@@ -22,6 +22,8 @@ import EventEmitter from './events';
 import { isIOS } from './deviceInfo';
 
 export const saveLastLocalAuthenticationSession = async (server: string, serverRecord?: TServerModel): Promise<void> => {
+	const timesync: number = (await RocketChat.getServerTimeSync(server)) ?? 0;
```

Comment:
It's awkward to call this request again here.
`localAuthenticate()` already calls it.